Incorporate feedback about actions and functions (#572)
cap-js/docs: java/consumption-api.md

ApplicationService adminService = catalog.getService(ApplicationService.class, "AdminService");
```

> As of version 2.4.0, the [CAP Java SDK Maven Plugin](./development/#cds-maven-plugin) is capable of generating specific interfaces for services in the CDS model. These service interfaces also provide Java methods for actions and functions, which allow direct access to the action's or function's parameters. These specific interfaces can also be used to access the service:

Reviewer: ```suggestion
As of version 2.4.0, the [CAP Java SDK Maven Plugin](./development/#cds-maven-plugin) is capable of generating specific interfaces for services in the CDS model. These service interfaces also provide Java methods for actions and functions, which allows easily calling actions and functions with their parameters. These specific interfaces can also be used to get access to the service:
```

}
```

Instead of the generic service interface, also the more specific service interfaces can be injected:

```java
@Component
public class EventHandlerClass implements EventHandler {

    @Autowired
    private PersistenceService db;

    @Autowired
    private AdminService adminService;

}
```
::: tip

> For the injection of specific service interfaces the annotation `@Qualifier` is not required.

Reviewer: ```suggestion
For the injection of specific service interfaces the annotation `@Qualifier` is usually not required.
```

If there are multiple instances of the same CDS service at runtime a qualifier might still be required. But this is not often the case.

}
```

The `cds-maven-plugin` generates specific interfaces for services in the CDS model. These service interfaces provide Java methods for each action and function, which can be used to trigger the corresponding event.

> For further details, see [Trigger Action or Function](./application-services#trigger-action-or-function).

Author: I will schedule a meeting to discuss what shall be removed or changed. I'm not sure what you exactly mean.
